burner-wallet-plugin/tokenbridge-bw-exchange/src/utils/utils.ts

```diff
@@ -51,6 +51,9 @@ export const isBridgeContract = async (contract: Contract, allowedModes?: string
     }
     return allowedModes.includes(mode)
   } catch (e) {
-    return false
+    if (e.message.includes("Returned values aren't valid")) {
```

Comment:
What is your advise to verify this functionality with the Infura or xDai rpc endpoints?

Reply:
Well, I would suggest to add an additional `console.log` statement here, which will print out the `e.message`, so that we can see what kind of errors get in different situations.
In my local experiments, the error starting from `Returned values aren't valid` happened only when the callee was either an EOA or some non-bridge contract. Also, seems that this error is raised from the `web3js` library, so the error message should not depend on the particular JSON-RPC implementation.